locale/pt-br/texts.po

```diff
@@ -698,92 +700,267 @@ msgstr "Sua transferência de **${ this.amountAndToken }** foi confirmada"
 "If you unregister this token **you won't be able to execute operations with "
 "it**, unless you register it again."
 msgstr ""
 "Se você desregistrar esse token **você não conseguirá mais executar "
 "operações com ele**, a não ser que você o registre novamente."
 
-#: src/screens/UnregisterToken.js:121
+#: src/screens/UnregisterToken.js:120
 msgid ""
 "You won't lose your tokens, they will just not appear on this wallet anymore."
 msgstr ""
 "Você não irá perder os seus tokens, eles apenas não irão mais aparecer nessa "
 "wallet."
 
-#: src/screens/UnregisterToken.js:125
+#: src/screens/UnregisterToken.js:124
 #, javascript-format
 msgid "I want to unregister the token **${ tokenLabel }**"
 msgstr "Eu quero desregistrar o token **${ tokenLabel }**"
 
-#: src/screens/UnregisterToken.js:140
+#: src/screens/UnregisterToken.js:139
 msgid "Unregister token"
 msgstr "Desregistrar token"
 
-#: src/sagas/pushNotification.js:56
+#: src/screens/WalletConnect/WalletConnectList.js:33
+msgid "There was an error connecting. Please try again later."
+msgstr ""
+"Ocorreu um erro durante a conexão. Por favor, tente novamente mais tarde."
+
+#.  translator: Used when the QR Code Scanner is opened, and user will manually
+#.  enter the information.
+#: src/screens/WalletConnect/WalletConnectList.js:111
+msgid "Add"
+msgstr "Adicionar"
+
+#: src/screens/WalletConnect/WalletConnectList.js:117
+msgid "End session"
+msgstr "Encerrar sessão"
+
+#: src/screens/WalletConnect/WalletConnectList.js:117
+msgid "This will disconnect the session."
+msgstr "Isso irá desconectar a sessão."
+
+#: src/screens/WalletConnect/WalletConnectList.js:119
+msgid "End"
+msgstr "Encerrar"
+
+#: src/screens/WalletConnect/WalletConnectList.js:137
+msgid "Wallet Connect Sessions"
+msgstr "Sessões do Wallet Connect"
+
+#: src/screens/WalletConnect/WalletConnectManual.js:39
+msgid "Manual Connection"
+msgstr "Digitar"
+
+#: src/screens/WalletConnect/WalletConnectManual.js:45
+msgid "Wallet Connect URI"
+msgstr ""
+
+#: src/screens/WalletConnect/WalletConnectManual.js:57
+#: src/screens/WalletConnect/WalletConnectScan.js:34
+msgid "Connect"
+msgstr "Conectar"
+
+#.  translator: Used when the QR Code Scanner is opened, and user will manually
+#.  enter the information.
+#: src/screens/WalletConnect/WalletConnectScan.js:41
+msgid "Manual"
+msgstr "Digitar"
+
+#: src/screens/NetworkSettings/CustomNetworkSettingsScreen.js:16
+msgid "Custom Network Settings"
+msgstr "Configurações de Rede Personalizada"
+
+#: src/screens/NetworkSettings/CustomNetworkSettingsScreen.js:17
+msgid ""
+"Any token outside mainnet network bear no value. Only change if you know "
+"what you are doing."
+msgstr ""
+"Nenhum token fora da rede mainnet possui valor. Apenas mude a rede se você "
+"souber o que está fazendo."
+
+#: src/screens/NetworkSettings/CustomNetworkSettingsScreen.js:18
+#: src/screens/NetworkSettings/helper.js:4
+msgid "Updating custom network settings..."
+msgstr "Atualizando configurações de rede…"
```

Comment:
```suggestion
msgstr "Atualizando configurações de rede..."
```

Reply:
Was that a symbol for "..."? Was it on purpose?

Reply:
No. The localization tool should have captured my ligature glyph, somehow.

Reply:
Fixed: https://github.com/HathorNetwork/hathor-wallet-mobile/pull/361/commits/3930be656af6644b3c011bb00a2f51358eb2e5f5